app/frontend/src/components/ScoreBar.stories.tsx

```diff
@@ -0,0 +1,19 @@
+import { Meta, Story } from "@storybook/react";
+
+import ScoreBar from "./ScoreBar";
+
+export default {
+  title: "Components/Large/ScoreBar",
+  component: ScoreBar,
+} as Meta;
+
+const Template: Story<any> = (args) => (
+  <>
+    <ScoreBar {...args}>Community standing</ScoreBar>
```

Comment:
We might want to change text colours depending on if we are keeping this purple... the black text on purple makes the text impossible to read